pdc/apps/compose/views.py

```diff
@@ -1276,6 +1277,110 @@ def _clear(self, release_obj, args):
+        allowed_keys = list(keys) + arg_filter_map
+        extra_keys = set(data.keys()) - set(allowed_keys)
+        if extra_keys:
+            return Response({'detail': '%s keys are not allowed' % list(extra_keys)},
+                            status=status.HTTP_400_BAD_REQUEST)
+
+        if 'source_release_id' not in data:
+            return Response({'detail': 'Missing source_release_id'},
+                            status=status.HTTP_400_BAD_REQUEST)
+        source_release_id = data.pop('source_release_id')
+        if 'target_release_id' not in data:
+            return Response({'detail': 'Missing target_release_id'},
+                            status=status.HTTP_400_BAD_REQUEST)
+        target_release_id = data.pop('target_release_id')
+        try:
+            source_release = get_object_or_404(Release, release_id=source_release_id)
+        except Http404:
+            return Response({'detail': 'Source_release %s is not existed' % source_release_id},
+                            status=status.HTTP_404_NOT_FOUND)
+        try:
+            target_release = get_object_or_404(Release, release_id=target_release_id)
+        except Http404:
+            return Response({'detail': 'Target_release %s is not existed' % target_release_id},
+                            status=status.HTTP_404_NOT_FOUND)
+
+        kwargs = {'release__release_id': source_release.release_id}
+        for arg in arg_filter_map:
+            arg_data = request.data.get(arg)
+            if arg_data:
+                kwargs[arg] = arg_data
+
+        overrides_rpm = OverrideRPM.objects.filter(**kwargs)
+        if not overrides_rpm:
+            return Response({'detail': 'there is no overrides-rpm with source release'},
+                            status=status.HTTP_400_BAD_REQUEST)
+        results = []
+        for rpm in overrides_rpm:
+            orpm, created = OverrideRPM.objects.get_or_create(release=target_release,
+                                                              rpm_name=rpm.rpm_name,
+                                                              rpm_arch=rpm.rpm_arch,
+                                                              variant=rpm.variant,
+                                                              arch=rpm.arch,
+                                                              srpm_name=rpm.srpm_name,
+                                                              comment=rpm.comment
+                                                              )
+            if created:
+                results.append(orpm.export())
+                request.changeset.add('OverridesRPM', orpm.pk,
+                                      'null', json.dumps(orpm.export()))
+            else:
+                continue
+        if results:
+            return Response(status=status.HTTP_201_CREATED, data=results)
+        else:
+            return Response({'detail': 'Both releases contain the same overrides-rpm'},
```

Comment:
it seems if target overrides-rpm >= source overrides-rpm will go into this case. so the error message is not correct